### ISSUE TITLE
Añadir shuffling a CV

### ISSUE THREAD

**Issue by daviddiazvico** [daviddiazvico/git_practica_ml#4]
El score de CV sigue siendo insatisfactorio. El dataset de Boston Housing tiene
unos cuantos ouliers juntos, así que los scores de algunos de los folds son muy
bajos. Hay que hacer shuffling del dataset antes de la CV.

CHULETA
Primero hay que hacer PULL para actualizar el repositorio.
experiment.py debe quedar así:

#!/usr/bin/env python3

import numpy as np
from sklearn.model_selection import cross_validate, KFold

from dataset import load_dataset
from model import load_model

X, y = load_dataset()
estimator = load_model()
cv_scores = cross_validate(estimator, X, y, cv=KFold(n_splits=5, shuffle=True))
estimator.fit(X, y)
score = estimator.score(X, y)
print(f"Train score: {score}")
print(f"CV score: {np.mean(cv_scores['test_score'])} ({cv_scores['test_score']})")
